site/_release/0.14.1.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: default
+title: Apache Arrow 0.14.1 Release
+permalink: /release/0.14.1.html
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+# Apache Arrow 0.14.1 (22 July 2019)
+
+This is a major release covering more than 0 months of development.
```

Comment:
This reads a little oddly. Technically it's a minor release so I'm confused as to whether or not I need to upgrade. I've deployed 0.14.0. If this is indeed a *major* release should I update my deployment?

Reply:
Thanks for the catch, this was automatically generated!

Reply:
Yeah, the whole sentence should go. Replace it with a brief note about why we did the patch release (parquet and wheels fixes). That should also make clear who this release is relevant for. (cf. https://arrow.apache.org/release/0.11.1.html)

Reply:
The changelog is also missing.